src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,101 @@
+package duke;
+
+public class Parser {
+    private final TaskList DUKE_LIST;
```

Comment:
Is this supposed to be in camelCase? This syntax looks like it is for constants (`static final` fields).